Task2/src/app.ts

```diff
@@ -0,0 +1,19 @@
+import './assets/styles/app-styles.css';
+
+import 'zone.js';
```

Comment:
![capture](https://cloud.githubusercontent.com/assets/13553328/20866126/cae655ee-ba35-11e6-9baf-e6d621035b72.JPG)
If I remove this line I got this error.